src/drivers/uavcannode/Publishers/FlowMeasurement.hpp

```diff
@@ -80,6 +80,7 @@ class FlowMeasurement :
 			// rotate measurements in yaw from sensor frame to body frame
 			measurement.rate_gyro_integral[0] = optical_flow.delta_angle[0];
 			measurement.rate_gyro_integral[1] = optical_flow.delta_angle[1];
+			measurement.rate_gyro_integral[2] = optical_flow.delta_angle[2];
```

Comment:
unfortunately this would require to update https://github.com/dronecan/DSDL/blob/5d303e1a9eb9bce17bb58a2378c1666d1cef3665/com/hex/equipment/flow/20200.Measurement.uavcan#L2

Reply:
If we don't want to update it, we could change EKF2 to use its own gyro data for the Z axis

Reply:
We can't change that specific DSDL, but we could add a new one (including timestamp, etc).